molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -65,6 +65,7 @@
         log_driver: json-file
         command: "{{ item.command | default('bash -c \"while true; do sleep 10000; done\"') }}"
         privileged: "{{ item.privileged | default(omit) }}"
+        sysctls: "{{ item.sysctls | default(omit) }}"
```

Comment:
Thanks for this @f-roscher. I think we need to add this to the schema in https://github.com/ansible/molecule/blob/master/molecule/model/schema_v2.py and include some unit tests to assure that is validation as expected.

Reply:
@f-roscher Can you please address that request? I will be happy to help you merge this improvement once you also update the schema.